### PR TITLE
Ms rubocop implementation

### PR DESCRIPTION
**Pull request by scara82**
Add rubocop and fixed style offences

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -1,32 +1,19 @@
-#!/usr/bin/env ruby
```
Comment (elle) [inline bin/setup:1]: You need this line to be able to run this file as an executable

File: bin/setup [hunk 2]
```diff
@@ -1,32 +1,19 @@
...
 
-  # Install JavaScript dependencies if using Yarn
-  # system('bin/yarn')
-
-
-  # puts "\n== Copying sample files =="
-  # unless File.exist?('config/database.yml')
-  #   cp 'config/database.yml.sample', 'config/database.yml'
-  # end
-
```
Comment (elle) [inline bin/setup]: I would also keep some of these comments, like line 14-15.
But then again YMMV, whatever you prefer

Comment (elle) [reply bin/setup]: Also double quotes...

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +0,0 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
```
Comment (elle) [inline db/seeds.rb:7]: I would still keep this file, which is what we use for seeding production (or even local database with production data) -- data that we want to keep

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -2,8 +2,10 @@
 ENV["RAILS_ENV"] ||= "test"
 require File.expand_path("../../config/environment", __FILE__)
 
-abort("The Rails environment is running in production mode!") if Rails.env.production?
-require "rspec/rails"
+abort("The Rails environment is running in production mode!")
+if Rails.env.production?
+  require "rspec/rails"
+end
```
Comment (elle) [inline spec/rails_helper.rb]: This doesn't make sense. We don't want to run rspec in production.
We are also setting the Rails env specifically on line 1.
The `if` block should be around the `abort` call

File: db/seeds.rb (hunk 3 above)
Comment (scara82) [reply db/seeds.rb:7]: I just eliminated the comments, the file is still there :)

File: spec/rails_helper.rb (hunk 4 above)
Comment (scara82) [reply spec/rails_helper.rb]: Ok, I will remove this.

File: bin/setup (hunk 1 above)
Comment (scara82) [reply bin/setup:1]: 👍

**Pull request merged**